build-system/tasks/runtime-test.js

```diff
@@ -200,17 +203,9 @@ gulp.task('test', 'Runs tests',
   } else if (argv.randomize || argv.glob || argv.a4a) {
     var testPaths;
     if (argv.a4a) {
-      testPaths = [
-        'extensions/amp-a4a/**/test/**/*.js',
-        'extensions/amp-ad-network-*/**/test/**/*.js',
-        'ads/google/a4a/test/*.js'
-      ];
+      testPaths = config.a4aTestPaths;
     } else {
-      testPaths = [
-        'test/**/*.js',
-        'ads/**/test/test-*.js',
-        'extensions/**/test/**/*.js',
-      ];
+      testPaths = config.basicTestPaths;
     }
 
     var testFiles = [];
```

Comment:
And I believe this block can be replaced by:

```
const testFiles = glob.sync(testPaths[index]);
```

Reply:
discussed offline

Reply:
Never mind, I didn't see the loop around this line :(